src/BUILD

```diff
@@ -25,6 +25,7 @@ cc_library(
         "pieces.cpp",
         "square.cpp",
         "squares_board.cpp",
+        "split.cpp",
```

Comment:
`split.cpp` was missing from your "board" library, causing the following link error in Linux:

```
ben@lenovo: (link-fixups) /scratch/schach$ bazel test ...
INFO: Analysed 6 targets (5 packages loaded).
INFO: Found 3 targets and 3 test targets...
ERROR: /scratch/schach/test/BUILD:9:1: Linking of rule '//test:fen_test' failed (Exit 1)
bazel-out/local-fastbuild/bin/_solib_k8/libsrc_Slibfen.so: error: undefined reference to 'split(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, char)'
collect2: error: ld returned 1 exit status
INFO: Elapsed time: 12.821s, Critical Path: 3.36s
FAILED: Build did NOT complete successfully
//test:piece_test                                                     NO STATUS
//test:square_test                                                    NO STATUS

Executed 0 out of 3 tests: 1 fails to build and 2 were skipped.
```